NSS-Day-Cohort-49/react-nutshell-state-slingers
# HCR - Final Article Implementation and Styling

#### Changes Made
1. Modified file `ApplicationViews` to adjust wrapping.
1. Modified `Article.css` to add styling to Article components
1. Modified `ArticleCard` to add styling to friend user articles, and display edit/delete button for the user articles only.
1. Modified `ArticleList` to sort the entries newest to oldest, and display only user and friends articles
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout hcr-addlarticlestuff
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Logged-in user should see only user and friend articles.
    > Friends articles should be yellow with italicized text.
    > User should be able to edit articles they posted - this will update the "posted on" value, which should cause the articles 
        list to change sorting if editing not the most recent article.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
